loader/CMakeLists.txt

```
@@ -306,6 +306,9 @@ else()
     target_link_libraries(vulkan Vulkan::Headers)
     # Used to make alloca() and secure_getenv() available
     target_compile_definitions(vulkan PRIVATE _GNU_SOURCE)
+    if(${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
```

Minor style: other instances of `CMAKE_*` conditionals *in this project* don't use explicit `${...}` e.g., https://github.com/KhronosGroup/Vulkan-Loader/blob/61688e09fecf7f8b7749644110be9a806d4b0f4d/loader/CMakeLists.txt#L288

```suggestion
    if(CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
```

---

thank you, I've changed it and rebased!